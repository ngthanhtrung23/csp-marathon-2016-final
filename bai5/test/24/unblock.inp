.11456
.23456
.23006
778...
9.8ABB
9CCA..
