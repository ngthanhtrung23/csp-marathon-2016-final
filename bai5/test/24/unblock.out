30
1 L 1
3 U 1
6 D 1
A U 1
C R 3
8 D 1
A D 1
7 R 2
2 D 2
0 L 3
3 D 1
4 D 1
5 D 1
1 R 4
3 U 1
4 U 1
0 R 2
2 U 3
9 U 4
0 L 2
7 L 2
8 U 2
A U 2
B L 4
5 D 2
8 D 1
A D 1
C L 1
6 D 2
0 R 4

